preview/preview.go

```diff
@@ -0,0 +1,131 @@
+package preview
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	logger "github.com/ipfs/go-log"
+	"github.com/qri-io/dataset"
+	"github.com/qri-io/dataset/dsio"
+	"github.com/qri-io/qfs"
+)
+
+var (
+	log = logger.Logger("preview")
+)
+
+const (
+	// MaxNumDatasetRowsInPreview is the highest number of rows a dataset preview
+	// can contain
+	MaxNumDatasetRowsInPreview = 100
+	// MaxReadmePreviewBytes determines the maximum amount of bytes a readme
+	// preview can be. three bytes less than 1000 to make room for an elipsis
+	MaxReadmePreviewBytes = 997
+)
```

Comment:
`const`s pulled straight from `base/preview.go`

Reply:
Lovely. Most of the purpose of this package is to define what a preview is exactly. I'm hoping we can define these for all byte bounding

Reply:
In this PR or is this a note for the future?

Reply:
oh sorry, yes this is a note for the future. No need to change anything here